junit/src/main/resources/META-INF/services/org.junit.platform.launcher.LauncherSessionListener

```diff
@@ -0,0 +1 @@
+net.neoforged.fml.junit.JUnitService
```

Comment:
Any reason to specify the services both in the module-info and in META-INF? Do we expect users to launch this outside of the module path?